consultation_analyser/consultations/models.py

```diff
@@ -1,7 +1,51 @@
-# Create your models here.
+import uuid
 from django.db import models
 
 
-class Question(models.Model):
+class UUIDPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+
+
+class SlugPrimaryKeyBase(models.Model):
+    class Meta:
+        abstract = True
+
+    slug = models.CharField(primary_key=True, blank=False, max_length=256)
+
+
+class Consultation(SlugPrimaryKeyBase):
+    name = models.CharField(max_length=256, blank=False)
+
+
+class Section(SlugPrimaryKeyBase):
```

Comment:
One more q — wouldn't questions belong to sections, then sections to consultations?

Reply:
also the null here — do we ever want a section without a consultation?

Reply:
Discussion: agreed to change model to be consultation > section > question

Reply:
Left the nulls in as discussed (makes it easier to create fake data).

Reply:
Updated to have section as a foreign key on question, and consultation as foreign key on section.